.github/workfows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: build
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  run:
+    name: build
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          #- windows-latest
+          #- macos-latest
+        ocaml-compiler:
```

Comment:
is there a 5.0 compiler?

Reply:
It's not released yet. The latest version is [5.0.0~alpha0](https://discuss.ocaml.org/t/ocaml-5-0-zeroth-alpha-release/10026), available as of June.

Reply:
there is alpha1 too.
I mean if it is available for github actions

Reply:
OK, I found (was told) the right name. I'll update the PR in a bit.
I also found out why my main.yml wasn't triggering actions - I had a typo in the `workflow` folder name